explainability-service/manifests/opendatahub/default/trusty-deployment.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    app.openshift.io/vcs-url: <<unknown>>
```

Comment:
Is this automatically replaced by some script? Should we populate it?

Reply:
Ill defer to @ruivieira  on this. I just copied his yamls :-)